src/core/utils.ts

```diff
@@ -53,3 +53,16 @@ export const jsNumberForAddress = (address: string): number => {
   const seed = parseInt(addr, 16);
   return seed;
 };
+
+export const filterOperators = [
```

Comment:
Do we perhaps want some descriptive text with these? A few are obvious, but I don't even know what they all do.

Reply:
yeah I don't know either, it's a great point.

Reply:
this appears to be the place: https://github.com/hyperledger/firefly/blob/04cd7184e0562a3a5a5344b0430bf68cc76415b1/internal/apiserver/restfilter.go#L126